crates/uv/src/commands/python/install.rs

```diff
@@ -265,11 +295,20 @@ pub(crate) async fn install(
     };
 
     let Some(first_request) = requests.first() else {
-        if upgrade {
-            writeln!(
-                printer.stderr(),
-                "There are no installed versions to upgrade"
-            )?;
+        match upgrade {
+            PythonUpgrade::Enabled(PythonUpgradeSource::Upgrade) => {
+                writeln!(
+                    printer.stderr(),
+                    "There are no installed versions to upgrade"
+                )?;
+            }
+            PythonUpgrade::Enabled(PythonUpgradeSource::Install) => {
```

Comment:
Review note: This is reachable by creating an empty `.python_versions` file (unlike the branch above, which looks at installed Pythons)